arviz/data/io_pystan.py

```diff
@@ -623,6 +654,73 @@ def get_sample_stats(fit, warmup=False):
     return data, data_warmup
 
 
+def get_attrs(fit):
+    """Get attributes from PyStan fit object."""
+    attrs = {}
+
+    try:
+        attrs["args"] = [deepcopy(holder.args) for holder in fit.sim["samples"]]
+    except Exception as exp:  # pylint: disable=broad-except
+        _log.warning("Failed to fetch args from fit: %s", exp)
+    if "args" in attrs:
+        for arg in attrs["args"]:
+            if isinstance(arg["init"], bytes):
+                arg["init"] = arg["init"].decode("utf-8")
+    try:
+        attrs["inits"] = np.array([holder.inits for holder in fit.sim["samples"]])
+    except Exception as exp:  # pylint: disable=broad-except
+        _log.warning("Failed to fetch `args` from fit: %s", exp)
+
+    attrs["step_size"] = []
+    attrs["metric"] = []
+    attrs["inv_metric"] = []
+    for holder in fit.sim["samples"]:
+        try:
+            step_size = float(
+                re.search(
+                    r"step\s*size\s*=\s*([0-9]+.?[0-9]+)\s*",
+                    holder.adaptation_info,
+                    flags=re.IGNORECASE,
+                ).group(1)
+            )
+        except AttributeError:

```

Comment:
is attribute error what happens when `re.search` returns None, which has no `.group` attribute?

Reply:
Yes, my lazy solution. "comment" field should still have the raw value.